lab-ron/lib/storage.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+const storage = {};
+const pool = [];
+
+exports.fetchAll = function(note) {
+  if (!note) return Promise.reject(new Error('expected note'));
```

Comment:
This is good, but you're just returning the filenames, and you should be displaying the full contents of all notes.